src/algebraic_geometry/morphisms/universally_closed.lean

```diff
@@ -31,8 +31,8 @@ open category_theory.morphism_property
 open algebraic_geometry.morphism_property (topologically)
 
 /--
-A morphism of schemes `f : X ⟶ Y` is locally of finite type if for each affine `U ⊆ Y` and
-`V ⊆ f ⁻¹' U`, The induced map `Γ(Y, U) ⟶ Γ(X, V)` is of finite type.
+A morphism of schemes `f : X ⟶ Y` is universally closed if `X ×[Y] Y' ⟶ Y'` is a closed map
+for all base change `Y' ⟶ Y`.
```

Comment:
```suggestion
A morphism of schemes `f : X ⟶ Y` is universally closed if `X ×[Y] Y' ⟶ Y'` is a closed map
for every base change `Y' ⟶ Y`.
```
I'm not sure if it's appropriate to use "base change" here since the word usually refers to the morphism `X ×[Y] Y' ⟶ Y'` rather than `Y' ⟶ Y`.

Reply:
how about "base change *along* `Y' ⟶ Y`"?

Reply:
I saw your suggestion about using "along", which sounds a good idea to me! This may be better, but what you currently have is also good.
```suggestion
A morphism of schemes `f : X ⟶ Y` is universally closed if the base change `X ×[Y] Y' ⟶ Y'`
along every morphism `Y' ⟶ Y` is a closed map.
```

Reply:
@jcommelin Do you think my version (above) is better, or the current version (below)?
```
A morphism of schemes `f : X ⟶ Y` is universally closed if
every base change `X ×[Y] Y' ⟶ Y'` is a closed map.
```
I see you thumbed up my comment above but I'm not sure whether you think mine is better or agree that the current version is okay. Maybe you could just make a decision and (commit and) merge whichever version you think is better? Thanks!

Reply:
I didn't see the new suggestion. It has now become
```
A morphism of schemes `f : X ⟶ Y` is universally closed if the base change `X ×[Y] Y' ⟶ Y'`
along any morphism `Y' ⟶ Y` is (topologically) a closed map.
```

Reply:
Thanks!

maintainer merge